practice-app/backend/controllers/weatherNextDays.js

```diff
@@ -0,0 +1,105 @@
+const Response = require('../utils/response');
+const Request = require('../utils/request');
+/**
+ * Get the weather of Istanbul for the next 8 days from
+ * openweather api.
+ * Response: {
+ *    status: {
+ *        success: true,
+ *        code: 200
+ *    }, data: {
+ *        day1: {
+ *            minTemp: 18,
+ *            maxTemp: 26,
+ *            desc: 'broken clouds'
+ *        },
+ *        day2: {
+ *            minTemp: 17,
+ *            maxTemp: 27,
+ *            desc: 'broken clouds'
+ *        },
+ *        day3: {
+ *            minTemp: 19,
+ *            maxTemp: 26,
+ *            desc: 'overcast clouds'
+ *        },
+ *        day4: {
+ *            minTemp: 16,
+ *            maxTemp: 20,
+ *            desc: 'light rain'
+ *        },
+ *        day5: {
+ *            minTemp: 15,
+ *            maxTemp: 19,
+ *            desc: 'broken clouds'
+ *        },
+ *        day6: {
+ *            minTemp: 14,
+ *            maxTemp: 18,
+ *            desc: 'broken clouds'
+ *        },
+ *        day7: {
+ *            minTemp: 13,
+ *            maxTemp: 18,
+ *            desc: 'clear sky'
+ *        },
+ *    }
+ *}
+ */
+module.exports.weatherNextDays = function (request, response){
+    try {
+        Request.send({ // Sends request to openweather api
+            url: 'https://api.openweathermap.org/data/2.5/onecall?lat=41.008240&lon=28.978359&units=metric&exclude=current,minutely,hourly&appid=' + process.env.OPENWEATHER_API_KEY,
```

Comment:
Latitude and Longitude are hardcoded but it is OK, in terms of the goals of the assignment, I think. Also, the coordinates are locating Istanbul, which shows they are not chosen randomly.